app/vmui/packages/vmui/src/pages/CardinalityPanel/CardinalityTips/index.tsx

```diff
@@ -0,0 +1,112 @@
+import { TipIcon } from "../../../components/Main/Icons";
+import React, { FC } from "preact/compat";
+import { ReactNode } from "react";
+import "./style.scss";
+
+const Link: FC<{ href: string, children: ReactNode }> = ({ href, children }) => (
+  <a
+    href={href}
+    className="vm-link vm-link_colored"
+  >
+    {children}
+  </a>
+);
+
+const TipCard: FC<{ title?: string, children: ReactNode }> = ({ title, children }) => (
+  <div className="vm-cardinality-tip">
+    <div className="vm-cardinality-tip-header">
+      <div className="vm-cardinality-tip-header__tip-icon"><TipIcon/></div>
+      <h4 className="vm-cardinality-tip-header__title">{title || "Tips"}</h4>
+    </div>
+    <p className="vm-cardinality-tip__description">
+      {children}
+    </p>
+  </div>
+);
+
+export const TipDocumentation: FC = () => (
+  <TipCard title="Cardinality explorer">
+    <h6>Helpful for analyzing VictoriaMetrics TSDB data</h6>
+    <ul>
+      <li>
+        <Link href="https://docs.victoriametrics.com/#cardinality-explorer">
+          Cardinality explorer documentation
+        </Link>
+      </li>
+      <li>
+        See the <Link href="https://victoriametrics.com/blog/cardinality-explorer/">
+        example of using</Link> the cardinality explorer
+      </li>
+    </ul>
+  </TipCard>
+);
+
+export const TipHighNumberOfSeries: FC = () => (
+  <TipCard title="Metrics with a high number of series">
+    <ul>
+      <li>
+        You can to identify and eliminate labels with frequently changed values to reduce its&nbsp;
+        <Link href='https://docs.victoriametrics.com/FAQ.html#what-is-high-cardinality'>cardinality</Link>&nbsp;and&nbsp;
+        <Link href='https://docs.victoriametrics.com/FAQ.html#what-is-high-churn-rate'>high churn rate</Link>
+      </li>
+      <li>
+         You can find unused metrics and <Link href='https://docs.victoriametrics.com/relabeling.html'>drop entire metrics</Link>
+      </li>
+      <li>
+          You can replace a large number of underlying series with <Link href='https://docs.victoriametrics.com/stream-aggregation.html'>streaming aggregation</Link>
+      </li>
+    </ul>
+  </TipCard>
+);
+
+export const TipHighNumberOfValues: FC = () => (
+  <TipCard title="Labels with a high number of unique values">
+    <ul>
+      <li>You can decrease its number of values</li>
+      <li>You can drop this label entirely</li>
+      <li>If you still this label with its values, could you store it in a log file?</li>
```

Comment:
```suggestion
      <li>For volatile label values (such as URL path, user session, etc.) consider printing them to the log file instead of adding to time series</li>
```